CONTRIBUTING.md

```diff
@@ -107,7 +107,7 @@ nx e2e e2e-cli -t versions # I often add qqqq to my test name so I can use -t qq
 To build Nx on Windows, you need to use WSL.
 
 - Run `yarn install` in WSL. Yarn will compile several dependencies. If you don't run `install` in WSL, they will be compiled for Windows.
-- Run `yarn test` and other commands in WSL.
+- Run `yarn jest` and other commands in WSL.
```

Comment:
we should replace it with "nx affected --target=test" to match the non-wsl docs

Reply:
Let's stick with `nx affected --target=xyz` :+1: It's more generic.

Reply:
Done